twilio/base/obsolete.py

```diff
@@ -21,3 +21,23 @@ def new_func(*args, **kwargs):
         )
 
     return new_func
+
+
+def deprecated_method(new_func=None):
```

Comment:
Wonder why `warnings` was already imported. What else used to be in this file?

Reply:
Just the `ObsoleteException` definition and the `@obsolete_class` decorator. `warnings` wasn't being used but the import hadn't been removed.